charts/pomerium/templates/tls-secrets.yaml

```diff
@@ -143,6 +150,12 @@ data:
   {{ template "pomerium.authorize.tlsSecret.certName" . }}: {{ .Values.authorize.tls.cert | b64enc }}
   {{ template "pomerium.authorize.tlsSecret.keyName" . }}: {{ .Values.authorize.tls.key | b64enc }}
 {{-     end }}
+{{-     if and (not .Values.cache.existingTLSSecret) .Values.cache.tls.cert .Values.cache.tls.key }}
+---
+{{ template "pomerium.cache.tlsSecretObject" . }}
```

Comment:
You'll need a block at the top of this file for the TLS secret hooks

Reply:
Added